benchmarks/pipe.py

```diff
@@ -577,7 +607,10 @@ def bench_mpi(args):
 )
 parser.add_argument("--use_synthetic_data", default=True, help="Uses synthetic data for a sample training run.")
 parser.add_argument(
-    "--model_name", default="seq_pred", choices=["seq_pred", "transformer"], help="Model used to benchmark pipe."
+    # TODO(anj-s): In the process of adding more models and hence the requirement for a flag.
+    "--model_name",
+    default="lm",
```

Comment:
should this be a `choice` type for argparse?

Reply:
Yes, I was planning to add the `choices` parameter when adding support for running a second model.